runtime/src/iree/hal/local/executable_library.h

```diff
@@ -401,6 +401,18 @@ typedef struct iree_hal_executable_src_loc_v0_t {
   const char* path;
 } iree_hal_executable_src_loc_v0_t;
 
+// Source file contents for a dispatch function indicating what was used to
```

Comment:
as mentioned on the compiler side, this is the wrong granularity. we should just have unique files that are referenced by the existing src_locs - there's not a 1:1 mapping of exports to files.